doc/source/cluster/kubernetes/user-guides/webhooks.md

```diff
@@ -0,0 +1,88 @@
+(kuberay-webhooks)=
+
+# Webhooks
+
+This guide explains how to enable KubeRay's [K8s admission webhooks]. KubeRay supports the
+following webhooks.
+
+* RayCluster validating admission webhook
+
+## Quickstart
+
+### Step 1: Create a Kubernetes cluster with Kind
+
+```sh
+kind create cluster --image=kindest/node:v1.24.0
+```
+
+### Step 2: Install the KubeRay operator with webhooks
+
+Deploy the KubeRay operator with the [Helm chart repository](https://github.com/ray-project/kuberay-helm).
```

Comment:
@kevin85421 do we need to support installing webhooks with helm? I noticed helm is the suggested installation method in these docs.

Reply:
Most users install the KubeRay operator using Helm, so supporting Helm is beneficial if we want to promote webhooks. Initially, we can provide instructions in this doc for installing the webhook using `kubectl` or `kustomize` commands, while installing the KubeRay operator via Helm. We can update the doc after we add webhook support to the Helm chart.